velox/exec/TableScan.cpp

```diff
@@ -148,6 +148,13 @@ RowVectorPtr TableScan::getOutput() {
 
       if (connectorSplit->dataSource) {
         ++numPreloadedSplits_;
+        if (auto iter = std::find(
+                preloadingSplits_.begin(),
+                preloadingSplits_.end(),
+                connectorSplit);
+            iter != preloadingSplits_.end()) {
+          preloadingSplits_.erase(iter);
```

Comment:
Is this belong to the fix? Thanks.

Reply:
Yes, if `move()` is called, we don't needto call `close()`.